test/e2e.test.mjs

```diff
@@ -6,10 +6,81 @@ import {
   getValidator,
 } from "../dist/esm/util.js";
 import { LocalTableland } from "../dist/esm/main.js";
+import { startMockServer, stopMockServer } from "./util.mjs";
 
 const expect = chai.expect;
 const localTablelandChainId = 31337;
 
+describe("Validator and Chain startup and shutdown", function () {
+  let server;
+  const lt = new LocalTableland({ silent: true });
+
+  this.timeout(20000); // Starting up LT takes 3000-7000ms; shutting down takes <10-10000ms
```

Comment:
i observed that on my machine, the LT startup usually takes 3-7 seconds, and shutdown was <10ms to 10 seconds. i'm not sure why the variance. 

e.g., one test would log:
```
  Validator and Chain startup and shutdown
lt start took 4313 milliseconds.
    ✔ fails to start due to being blocked on port 8545 (11328ms)
stopMockServer took 0 milliseconds.
lt.shutdown took 8 milliseconds.
lt start took 6968 milliseconds.
    ✔ successfully starts using retry logic after initially blocked (6968ms)
stopMockServer took 0 milliseconds.
lt.shutdown took 7 milliseconds.
```
and running it again would log:
```
  Validator and Chain startup and shutdown
lt start took 3371 milliseconds.
    ✔ fails to start due to being blocked on port 8545 (10384ms)
stopMockServer took 0 milliseconds.
lt.shutdown took 9 milliseconds.
lt start took 5846 milliseconds.
    ✔ successfully starts using retry logic after initially blocked (5847ms)
stopMockServer took 0 milliseconds.
lt.shutdown took 10021 milliseconds.
```